source/chapters/library.rst

```diff
@@ -835,18 +835,20 @@ play. See :ref:`getting-started-analyze-library` for details.
 
 .. _library-third-party:
 
-iTunes, Traktor, Rhythmbox, Banshee - Using external libraries
-==============================================================
+Using libraries from other software
+===================================
 
 .. sectionauthor::
    S.Brandt <s.brandt@mixxx.org>
+   Ukpai Ugochi <outreachy applicant>
```

Comment:
You can put your email address here if you want.

Reply:
done!